packages/react/src/components/searchBar/tests/SearchBar.spec.tsx

```diff
@@ -1,124 +1,105 @@
-    });
 
-    it('should not throw on render', () => {
+    it('does not throw', () => {
         expect(() => {
-            searchBarPropsScenarios.forEach((props) => shallow(<SearchBar {...props} />));
+            searchBarPropsScenarios.forEach((props) => render(<SearchBar {...props} />));
         }).not.toThrow();
     });
 
-    it('should have a container div with the search-bar class and without the search-bar-loading and search-bar-disabled classes by default', () => {
-        const containerDiv = shallow(<SearchBar {...requiredProps} />)
-            .find('div')
-            .first();
+    it('render the search bar', () => {
+        render(<SearchBar {...requiredProps} />);
 
-        expect(containerDiv.hasClass('search-bar')).toBe(true);
-        expect(containerDiv.hasClass('search-bar-loading')).toBe(false);
-        expect(containerDiv.hasClass('search-bar-disabled')).toBe(false);
+        expect(screen.getByRole('textbox')).toBeInTheDocument();
+        expect(screen.getByRole('img', {name: 'search icon'})).toBeInTheDocument();
     });
 
-    it('should have a container div with search-bar-loading class when searching is passed as prop', () => {
-        const containerDiv = shallow(<SearchBar {...requiredProps} searching />)
-            .find('div')
-            .first();
+    it('renders a loading animation when is searching', () => {
+        render(<SearchBar {...requiredProps} searching />);
 
-        expect(containerDiv.hasClass('search-bar-loading')).toBe(true);
+        expect(screen.getByRole('alert')).toBeInTheDocument();
+        expect(screen.queryByRole('img', {name: 'search icon'})).not.toBeInTheDocument();
     });
 
-    it('should have a container div with search-bar-disabled class when disabled is passed as prop', () => {
-        const containerDiv = shallow(<SearchBar {...requiredProps} disabled />)
-            .find('div')
-            .first();
+    it('render a disabled search box if disabled is true', () => {
+        render(<SearchBar {...requiredProps} disabled />);
 
-        expect(containerDiv.hasClass('search-bar-disabled')).toBe(true);
+        expect(screen.getByRole('textbox')).toBeDisabled();
     });
 
-    it('should add the extra container classes if passed as prop', () => {
+    it('render the search with extra classes', () => {
         const containerDiv = shallow(<SearchBar {...requiredProps} containerClassNames="extra-class" />)
             .find('div')
             .first();
 
-        expect(containerDiv.hasClass('extra-class')).toBe(true);
+        expect(containerDiv.prop('className')).toContain('extra-class');
     });
 
-    it('should have an input inside the div container with the search-bar-input class by default', () => {
-        const component = shallow(<SearchBar {...requiredProps} />);
+    it('render the input of the search bar with extra classes', () => {
+        const component = shallow(<SearchBar {...requiredProps} inputClassNames="extra-class" />)
```

Comment:
I might be wrong, but I think you could use RTL instead of Enzyme for this test (and the one prior). I don't know if it would work, but maybe switch to this?
```suggestion
        render(<SearchBar {...requiredProps} inputClassNames="extra-class" />)
        
        expect(screen.getByRole('textbox').toHaveClass('extra-class');
```

Is there some instances we should use Enzyme?